evilwallet/output_manager.go

```diff
@@ -259,6 +260,42 @@ func (o *OutputManager) AwaitOutputToBeAccepted(outputID iotago.OutputID, waitFo
 	return accepted
 }
 
+func (o *OutputManager) AwaitAddressUnspentOutputToBeAccepted(addr *iotago.Ed25519Address, waitFor time.Duration) (outputID iotago.OutputID, output iotago.Output, err error) {
```

Comment:
this and all other await functions could be moved to utils and don't need to be part of outputManager

Reply:
yeah, I have another pr for this.